common/filestore_utils.py

```diff
@@ -30,13 +30,11 @@ def _using_gsutil():
 if _using_gsutil():
     from common import gsutil as filestore_utils_impl
 else:
-    # When gsutil is not used in the context, here it should use local_utils.
+    # When gsutil is not used in the context, it should use local_filestore.
     # TODO(zhichengcai): Implement local_filestore.py and import it here.
     from common import gsutil as filestore_utils_impl
```

Comment:
Why aren't we importing `local_filestore` here?

Reply:
There are some problems. I move the integration to #422. Please take a look.
In fact, the code in #405 is buggy. _using_gsutil always return true because the importing happens earlier than environment setting.

Reply:
Is this PR closed then?
Also, I think only when `run_experiment` is called is this code called first, otherwise the filestore should be set first.

Reply:
Let's not change `filestore_utils` in this PR. This PR focuses on `local_filestore` itself and its unit tests. Let's discuss `filestore_utils` in #422, where I change its implementation.